samples/embedded/README.md

```diff
@@ -0,0 +1,66 @@
+# Embedded Example
+
+This example shows how to
+
+1. Build a PyTorch functional model for training
+2. Import that model into IREE's compiler
+3. Compile that model to an IREE VM bytecode module
+4. Load the compiled module using IREE's high level runtime C API into a
+   lightweight program
+5. Train the loaded model
+
+This example was built with the goal of allowing you to be able to build it
+outside this repo in your own project with minimal changes.
+
+The weights for the model are stored in the program itself and updated in
+memory. This can be modified to be stored however you see fit.
+
+## Running the Example
+
+Install `iree-torch` and other dependencies necessary for this example.
+[iree-torch](git@github.com:iree-org/iree-torch.git) provides a number of
+convenient wrappers around `torch-mlir` and `iree` compilation:
+
+```shell
+pip install -f https://iree-org.github.io/iree/pip-release-links.html iree-compiler
+pip install -f https://llvm.github.io/torch-mlir/package-index/ torch-mlir
+pip install git+https://github.com/iree-org/iree-torch.git
+pip install scikit-learn
```

Comment:
We usually recommend doing Python work in a venv (see the tip here: https://iree-org.github.io/iree/bindings/python/#prerequisites), though that does make the instructions more complex.

Reply:
Added a note.